server/events/models.py

```diff
@@ -0,0 +1,77 @@
+from datetime import timedelta
+from django.utils import timezone
+from django.contrib.auth import models as auth_models
+from django.db import models
+
+# Create your models here.
+class Event(models.Model):
+    """ A representation of an ACC event.
+
+    Attributes:
+        name = The name of the event.
+        creator = The User who created the event.
+        description = A brief description of the event.
+        location = Where the event is located.
+        start_datetime = The starting datetime for the event.
+        end_datetime = The ending datetime for the event.
+    """
+    name = models.CharField(max_length=50, blank=False)
+    creator = models.ForeignKey(auth_models.User, on_delete=models.deletion.PROTECT)
+    description = models.TextField(max_length=255)
+    location = models.CharField(max_length=255)
+    start_datetime = models.DateTimeField(default=timezone.now())
+    end_datetime = models.DateTimeField(default=timezone.now())
+
+    @property
+    def duration(self) -> timedelta:
```

Comment:
If the function's just a simple `return` statement, don't worry about writing a function definition.

Reply:
Also, `timedelta` was never imported. I'm not sure if this is ignored by the Python interpreter or not (I bet it's ignored), but for consistency, go ahead and `import datetime` at the top.

Reply:
fixed